packages/devtools_app/lib/src/logging/flutter/logging_screen.dart

```diff
@@ -357,6 +362,33 @@ class _MessageColumn extends LogMessageColumn
           ),
         ],
       );
+    } else if (data.kind == 'stdout') {
+      // TODO(helin24): Move this to logging controller when dart:html is removed.
+      const Color color = Color.fromARGB(0xff, 0x00, 0x91, 0xea);
+
+      final RichText richText = RichText(
+        text: TextSpan(
+          children: maybeConvertToAnsiText(
+              // TODO(helin24): Recompute summary length considering ansi codes?
+              //  The current summary is generally the first 200 chars of details.
+              getDisplayValue(data),
+              fixedFontStyle(context)),
+        ),
+        overflow: TextOverflow.ellipsis,
+      );
+
+      return Row(
+        children: <Widget>[
+          richText,
+          Flexible(
+            child: Container(
+              height: 12.0,
+              width: 0.0,
```

Comment:
is this width zero container needed?

Reply:
I don't really understand this param - if i remove `width` entirely, there are blue bars that show up to the right:
<img width="804" alt="Screen Shot 2020-04-23 at 9 32 52 AM" src="https://user-images.githubusercontent.com/6379305/80124721-83e0b200-8545-11ea-90e4-b198939e8e31.png">

Reply:
you don't want a blue bar. The blue bar is intended only for messages visualizing progress.

Reply:
Oh I misunderstood! Yes, I can remove this container.